src/SFA.DAS.ApprenticeCommitments.Web/Startup/ServicesStartup.cs

```diff
@@ -11,13 +13,28 @@ namespace SFA.DAS.ApprenticeCommitments.Web.Startup
     public static class ServicesStartup
     {
         public static IServiceCollection RegisterServices(
-            this IServiceCollection services)
+            this IServiceCollection services,
+            IWebHostEnvironment environment)
         {
             services.AddTransient<ApprenticeApi>();
             services.AddTransient<AuthenticatedUserClient>();
             services.AddTransient<ISimpleUrlHelper, AspNetCoreSimpleUrlHelper>();
             services.AddScoped<ITimeProvider, UtcTimeProvider>();
             services.AddTransient<IMenuVisibility, MenuVisibility>();
+            services.AddDomainHelper(environment);
+
+            return services;
+        }
+
+        private static IServiceCollection AddDomainHelper(this IServiceCollection services, IWebHostEnvironment environment)
+        {
+            var domain = ".localhost";
+            if (!environment.IsDevelopment())
+            {
+                domain = ".apprenticeships.education.gov.uk";
+            }
+
+            services.AddSingleton<DomainHelper>(x => new DomainHelper(domain));
```

Comment:
Not entirely sure, but I think this can be simplified
```suggestion
            services.AddSingleton(new DomainHelper(domain));
```